features/support/cucumber_rails_helper.rb

```diff
@@ -13,6 +13,7 @@ def install_cucumber_rails(*options)
     add_gem 'selenium-webdriver', '~> 3.11', group: :test
     add_gem 'rspec-expectations', '~> 3.7', group: :test
     add_gem 'database_cleaner', '>= 1.8.0', group: :test unless options.include?(:no_database_cleaner)
+    add_gem 'database_cleaner-active_record', '>= 2.0.0.beta2', group: :test if options.include?(:database_cleaner_active_record)
```

Comment:
Is this beta copy the best we have (This helper is used for everyone who uses the gem). Do we have any idea how close to a stable 2.x revision we are?

Reply:
Yes, this issue is the last blocker for releasing v2.0.0 final. So, as soon as this PR is merged, and out in the wild in a cucumber-rails gem release, I will cut the final v2.0.0 release of database_cleaner and its family of adapter gems.

Reply:
If we could coordinate a simultaneous release, we could change this to `>= 2.0.0`

Reply:
I'm confused, how is this helper used by everyone using this gem? This file is shipped with the gem, but its folder (`features`) is not exposed as a $LOAD_PATH directory. I thought these were just internal helpers for testing `cucumber-rails`.